src/lib/WriteThroughCache.ts

```diff
@@ -4,7 +4,31 @@ import { LocalCache } from './LocalCache';
 
 
 /**
- * Write-through cache, using Redis and a local LRU cache.
+ * Write-through cache, using Redis and a local LRU cache with aligned TTLs.
+ *
+ * **WARNING** if using this in a distributed app where cache consistency matters!
+ * Consider this case of an app with several servers/instances using WriteThroughCache:
+ *  1. Instance I1 sets key/value foo: bar (both local & redis)
+ *  2. Instance I2 gets value foo, populating its local cache with "bar"
+ *  ----- At that moment, LocalCaches of I1 & I2 are aligned, foo: bar
+ *  3. Instance I1 deletes key foo (both local & redis)
+ *  ----- At that moment, LocalCaches of I1 & I2 are *mis*aligned about foo!
+ *        - I1 has nothing
+ *        - I2 has "bar"
+ *
+ * -> This is fixable, e.g. using Redis pub/sub, to let clients subscribe to
+ *    set/del events, and react with an eviction from their LocalCache.
+ *    In the context we cachette maintainers got bitten by this, it made sense
```

Comment:
Are you sure? This sounds funny when I read it

```suggestion
 *    We cachette maintainers got bitten by this, and it made sense
```

Reply:
👍 was weird, I tend to over-omit some liaison words. Fixed in 5280a24.